.github/workflows/release.yaml

```diff
@@ -0,0 +1,69 @@
+name: Release
```

Comment:
I find it useful to see the latest released versions in the list of workflow runs.
We could use something like:
```
run-name: release ${{ inputs.tag }}
```